emperor/flat_plots.py

```diff
@@ -0,0 +1,346 @@
+from matplotlib import use
+use('Agg')  # noqa
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_3x3(ordination, offset, mapping, field, colors, null_kw=None,
```

Comment:
@wasade I will take over making these changes to convert plot_3x3 into something like the API I described above. Having this layout indeed fits your usecase, but is too specific.